docs/user/gcp/install_upi.md

```diff
@@ -0,0 +1,507 @@
+    ignition: '<master_ignition>'
+    image_name: '<image_name>'
+    machine_type: '<machine_type>'
+    root_volume_size: '128'
+    service_account_email: '<service_account_email>'
+```
+- infra_id: the infra_id from above (ie openshift-xxxx)
+- region: the region to deploy the cluster into (ie us-east1)
+- control_subnet: the api path to the control subnet
+  - `gcloud compute networks subnets describe ${INFRA_ID}-master-subnet --format json | jq -r .selfLink`
+- ignition: the contents of the master.ign file
+- image_name: the api path to the RHCOS image
+  - `gcloud compute images describe ${INFRA_ID}-rhcos-image --format json | jq -r .selfLink`
+- machine_type: The machine type of the instance (ie n1-standard-4)
+- service_account_email: the email address for the master service account created above
+  - `gcloud iam service-accounts list | grep "^${INFRA_ID}-master-node " | awk '{print $2}'`
+
+Create the deployment using gcloud.
+
+```
+gcloud deployment-manager deployments create ${INFRA_ID}-control-plane --config 05_control_plane.yaml
+```
+
+The templates do not manage dns entries due to limitations of Deployment
+Manager, so we must add the etcd entries manually
+
+```console
+export MASTER0_IP=`gcloud compute instances describe ${INFRA_ID}-master-0 --zone ${REGION}-a --format json | jq -r .networkInterfaces[0].networkIP`
+export MASTER1_IP=`gcloud compute instances describe ${INFRA_ID}-master-1 --zone ${REGION}-b --format json | jq -r .networkInterfaces[0].networkIP`
+export MASTER2_IP=`gcloud compute instances describe ${INFRA_ID}-master-2 --zone ${REGION}-c --format json | jq -r .networkInterfaces[0].networkIP`
+if [ -f transaction.yaml ]; then rm transaction.yaml; fi
+gcloud dns record-sets transaction start --zone ${INFRA_ID}-private-zone
+gcloud dns record-sets transaction add "${MASTER0_IP}" --name etcd-0.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 60 --type A --zone ${INFRA_ID}-private-zone
+gcloud dns record-sets transaction add "${MASTER1_IP}" --name etcd-1.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 60 --type A --zone ${INFRA_ID}-private-zone
+gcloud dns record-sets transaction add "${MASTER2_IP}" --name etcd-2.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 60 --type A --zone ${INFRA_ID}-private-zone
+gcloud dns record-sets transaction add \
+  "0 10 2380 etcd-0.${CLUSTER_NAME}.${BASE_DOMAIN}." \
+  "0 10 2380 etcd-1.${CLUSTER_NAME}.${BASE_DOMAIN}." \
+  "0 10 2380 etcd-2.${CLUSTER_NAME}.${BASE_DOMAIN}." \
+  --name _etcd-server-ssl._tcp.${CLUSTER_NAME}.${BASE_DOMAIN}. --ttl 60 --type SRV --zone ${INFRA_ID}-private-zone
+gcloud dns record-sets transaction execute --zone ${INFRA_ID}-private-zone
+```
+
+## Monitor for `bootstrap-complete` and Initialization
+
+```console
+$ openshift-install wait-for bootstrap-complete
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.12.4+c53f462 up
+INFO Waiting up to 30m0s for the bootstrap-complete event...
+```
+## Pivot load balancers to masters
```

Comment:
nit: The installer docs aren't all that consistent, and I prefer sentence case like this myself for headers, but your header right above ends with `and Initialization".  Probably pick either Title Case or Sentence case and make this file internally consistent.

Reply:
I used Sentance case, and reworded some of the sentences to make more sense.